charts/rancher-istio/1.5.1000/Chart.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: v1
+appVersion: 1.5.9
```

Comment:
Do `appVersio`/`version` need to be adjusted for the new version?

Reply:
are we adding support for istio 1.5.10 or are we making changes to istio 1.5.9? If we are making changes to 1.5.9, the Version on line 17 needs updated and we should be using 1.5.901, not 1.5.1000.

Reply:
We are changing 1.5.9 ill fix the versions